src/java/org/apache/cassandra/tools/CompactionLogAnalyzer.java

```diff
@@ -117,6 +117,8 @@
         int bucketsAboveT;
         // number of buckets above T*T sstables (excl compacting)
         int bucketsAboveT2;
+        // value of dynamic parameter W
```

Comment:
We use the term "scaling parameter" for W in the configuration. Let's adopt it here as well.